content/docs/command-reference/run.md

```diff
@@ -106,6 +106,11 @@ Relevant notes:
   also means that the stage command needs to recreate any directory structures
   defined as outputs every time its executed by DVC.
 
+- <abbr>Output</abbr> files or directories created by the stage commands which
+  are already in the workspace but missing from `dvc.yaml` (`outs` field) can be
```

Comment:
Same here. Just use the same text as much as possible since we already have a solution in content/docs/user-guide/how-to/add-output-to-stage.md because this version still doesn't clarify between files/dir "outputs" of stages and formally tracked _outputs_ (only the 2nd case should have `abbr` tags. Thanks

Reply:
Have updated it with almost same text as in `commit.md`

Reply:
why commit.md?

> we already have a solution in content/docs/user-guide/how-to/add-output-to-stage.md